test/support/utils.rb

```diff
@@ -3,6 +3,7 @@
 require 'pty'
 require 'timeout'
 require 'json'
+require_relative "../../lib/debug/client"
```

Comment:
This is required for:
- `DEBUGGER__.create_unix_domain_socket_name`
- `DEBUGGER__::Client`